Formula/proxysql.rb

```diff
@@ -0,0 +1 @@
+/Users/zac.bentley/Desktop/proxysql.rb
```

Comment:
❓

Reply:
fixing

Reply:
had a symlink in place instead of the actual formula, sorry. uploading real deal momentarily.